src/main/java/org/openmbee/mdk/emf/EMFExporter.java

```diff
@@ -161,7 +166,13 @@ private enum Processor {
         DOCUMENTATION_PRE(
                 (element, project, objectNode) -> {
                     //Todo: FIgure out how to read this stuff and parse the <a tags from MD
-                    // String doc_text = (String) Utils.getElementAttribute(element, Utils.AvailableAttribute.Documentation);
+                    String doc_text = (String) Utils.getElementAttribute(element, Utils.AvailableAttribute.Documentation);
```

Comment:
don't think this is completely implemented? the parsed doc isn't used anywhere, there's no mms-cf attr added, and should have a try catch surrounding it in case jsoup fails, also it'll lead to a diff since the output on mms will then be different to the model, think this should commented out and revisited later